easy_thumbnails/options.py

```diff
@@ -10,23 +7,22 @@ def __init__(self, *args, **kwargs):
         self._prepared_options = None
         super(ThumbnailOptions, self).__init__(*args, **kwargs)
```

Comment:
In order to be 100% close to python3, we can make the super call in modern way.
Like 
```
super().__init__(*args, **kwargs)```

Reply:
Done, also removed `object` inheritance from some classes